make/CMake_events.cmake

```diff
@@ -46,7 +46,9 @@ add_custom_command(
        ${SYSLOG_SRCS}
   VERBATIM # recommended: p260
   )
-if (WIN32)
+if (WIN32 AND NOT ("${CMAKE_GENERATOR}" MATCHES "MSYS Makefiles"))
+  # In MSYS2 we not define CMAKE_GENERATOR
```

Comment:
You mean CMAKE_MC_COMPILER

Reply:
And grammar fix -- missing "do": "we do not define"

Reply:
@derekbruening I mean this: https://cmake.org/cmake/help/v3.15/manual/cmake-generators.7.html?highlight=generator

Reply:
@derekbruening Fixed.